### DISCUSSION TITLE
Various appliances by Candy

### DISCUSSION THREAD

**Discussion by miguelfreitaspt** [ofalvai/home-assistant-candy#58] [Device support improvements]
Hello.

I have 3 smart appliances from Candy:

- an FCT825XL WIFI oven
(https://www.candy-home.com/pt_PT/fornos/33702981/fct825xl-wifi/)

- a CDPN 4S603PW/E dishwasher
(https://www.candy-home.com/pt_PT/maquinas-de-lavar-loica/32002168/cdpn-4s603pw-e/)

- an induction cooktop CTPJ644MCWIFI
(https://www.candy-home.com/pt_PT/placas/33802614/ctpj644mcwifi/)

I was able to integrate the dishwasher into HA, having access to the status and remaining time.
Is it possible to create buttons or automation for the machine to start washing?

I think the induction hob is not yet supported.

When I try to add the oven, my HA gives error and restarts automatically.
Is there anything I can do, to help?

Thank you for this fantastic integration. It is a shame that Candy itself does not support HA.

**Comment by ofalvai** [comment]
> Is it possible to create buttons or automation for the machine to start washing?

I haven't had time to reverse-engineer the command API yet, and I'm afraid the exact valid commands and parameters will be very device-specific. I think the first step will be to create a service that accepts the "raw" command parameters, such as this API endpoint: https://www.postman.com/candy-simply-fi/workspace/candy-simply-fi-s-public-workspace/request/11097163-d38eff97-9079-4640-b7ba-ae6eb0cfb764

> When I try to add the oven, my HA gives error and restarts automatically.
> Is there anything I can do, to help?

Please share the raw API response of your device using this tool: https://github.com/MelvinGr/CandySimplyFi-tool

**Comment by maxcanna** [comment]
@miguelfreitaspt I added the induction hob payload in https://github.com/ofalvai/home-assistant-candy/discussions/75

**Reply by miguelfreitaspt** [reply]
Great. What do i have to do to make it work, then?

**Reply by maxcanna** [reply]
We've to wait for @ofalvai to integrate it 🙏🏻
